src/sidebar/components/test/group-list-section-test.js

```diff
@@ -0,0 +1,70 @@
+'use strict';
+
+const angular = require('angular');
+const groupListSection = require('../group-list-section');
+const util = require('../../directive/test/util');
+
+describe('groupListSection', () => {
+  before(() => {
+    angular.module('app', []).component('groupListSection', groupListSection);
+  });
+
+  beforeEach(() => {
+    angular.mock.module('app', {});
+  });
+
+  const createGroupListSection = (
+    fakeSectionGroups,
+    fakeDisableOosGroupSelection
+  ) => {
+    const config = {
+      sectionGroups: fakeSectionGroups,
+    };
+    if (fakeDisableOosGroupSelection !== undefined) {
+      config.disableOosGroupSelection = fakeDisableOosGroupSelection;
+    }
+    return util.createDirective(document, 'groupListSection', config);
+  };
+
+  describe('isSelectable', () => {
+    [
+      {
+        description: 'always returns true if disableOosGroupSelection is false',
+        fakeDisableOosGroupSelection: false,
+        expectedIsSelectable: [true, true],
```

Comment:
The truth table is against fakeDisableOosGroupSelection and group.isScopedToUri. Note each of these cases tests two groups; [{ isScopedToUri}, { isScopedToUri: false}]. That should cover all the cases. I could break them up into cases that only test one type of group at a time if you think it's confusing as it's currently written?

Reply:
Oh right, sorry. I see. I was looking at an _output_ and thinking it was an input. You're testing three states of `fakeDisableOosGroupSelection`, so that's fine.